src/server/context.ts

```diff
@@ -672,7 +691,7 @@ export class ServerContext {
         baseRoute: toBaseRoute(ALIVE_URL),
         methods: {
           default: () => {
-            let timerId: number | undefined = undefined;
+            let timerId: Timer | undefined = undefined;
```

Comment:
bonus change 👀

Reply:
Can you change it back to `number`? There is no `Timer` type in Deno as their `setTimeout` return type is based on the browser API which returns a `number`. The `Timer` object only exists in node.

Reply:
looks like i need to do something with my VSCode settings as it was showing an error. I will revert this.

Reply:
Could be that the Deno vscode extension wasn't active or something.